modules/tinymce/src/core/main/ts/delete/CefDelete.ts

```diff
@@ -77,6 +78,13 @@ const backspaceDeleteRange = (editor: Editor, forward: boolean): Optional<() =>
       () => Optional.some(Fun.noop)
     );
   }
+
+  if (isCefAtEdgeSelected(editor)) {
+    return Optional.some(() => {
+      editor.selection.getRng().deleteContents();
+    });
+  }
```

Comment:
Why is this here? This Jira is only supposed to be able Home/End key navigation and this impacts deletion behaviour. I'm also concerned about this logic as `deleteContents()` doesn't clean up very well. This looks like it should be in a separate Jira and definitely needs a changelog entry.